test/integration/update_int_test.go

```diff
@@ -175,13 +175,6 @@ func (suite *UpdateIntegrationTestSuite) pollForUpdateInBackground(output string
 
 	suite.Require().Equal(len(resultLogfile), 2, "expected to have logfile in output %s", output)
 	logpath := strings.TrimSpace(resultLogfile[1])
-
-	for x := 0; x < 5; x++ {
-		if fileutils.FileExists(logpath) {
-			break
-		}
-		time.Sleep(1 * time.Second)
-	}
```

Comment:
Are you sure that this is not necessary anymore?  I think the mocked updates may install fast enough, but we might need this again once we add "real" tests.

Reply:
This wasn't here prior to my adding it to try and fix the logFile issue. Since the issue got resolved differently I removed it. Are you saying it should've had this all along?